Explore: Fix issues when loading and both graph/table are collapsed

marefr opened the pull request:
**What this PR does / why we need it**:
Removes the functionality of being able to collapse/expand the logs container.

**Which issue(s) this PR fixes**:
Fixes #17098 

**Special notes for your reviewer**:
Left the `showingLogs` ui property to not introduce a breaking change of parsing URL. Don't have any good idea how we can support removing of url parameters in a backward compatible way with the current solution.

marefr committed:
explore: remove collapsing of logs container functionality

davkal commented:
The original issue is true for Graphs and Tables as well. Can we fix this for those as well as part of this PR (keeping them collapsable)?

davkal commented on public/app/features/explore/Panel.tsx:
```diff
@@ -4,18 +4,27 @@ interface Props {
   label: string;
   loading?: boolean;
-  onToggle: (isOpen: boolean) => void;
+  isCollapsable?: boolean;
```
Spelling issue, also I'd propose `collapsible`, it's shorter. Or is there a benefit with the `is` prefix?

marefr replied:
Good catch. I went for is since `isOpen` was used. Will change :+1:

hugohaggmark reviewed (changes requested):
1. Change to a Prometheus datasource
2. Collapse both Table and Graph
3. Refresh page
4. Still can't run new queries

marefr committed:
explore: fix loading with graph and table collapsed

davkal commented:
Even on 56e7ed43ad I still can repro hugo's issue.

hugohaggmark commented:
Unfortunately I can still repro my issue.

marefr commented:
Hmm that's really weird. But with my latest commit when having graph and table collapsed and reloading page I swear I could expand graph and/or table and then queries are run. What am I missing?

marefr committed:
explore: fix so that reducer tests are run

On of the test used fit() instead of it() which had the consequence of
only 1 reducer test was executed and the rest skipped.
There was some failing tests that now is updated and now passes.

marefr committed:
explore: hide start page when dispatching runQueries action

This handles the special case when both graph and table are collapsed
and you reload the page then the start page should not be displayed.

marefr commented:
The issue was related to that the start page incorrectly was shown when reloading the page having graph/table collapsed. That should be fixed now together with actually running all reducer tests instead of only one :smile:

hugohaggmark commented:
LGTM!
